modules/core_project_factory/variables.tf

```diff
@@ -182,11 +182,17 @@ variable "disable_dependent_services" {
   type        = bool
 }
 
-variable "shared_vpc_enabled" {
-  description = "If shared VPC should be used"
+variable "enable_shared_vpc_service_project" {
+  description = "If this project should be attached to a shared VPC. If true, you must set shared_vpc variable"
   type        = bool
 }
 
+variable "enable_shared_vpc_host_project" {
+  description = "If this project is a shared VPC host project. If true, you must set shared_vpc variable. Default is false for compatibility"
```

Comment:
Actually if it's the host project we should *not* set `shared_vpc` variable.

Reply:
we must set... remember that shared_vpc == host project name.. that why we opened the issue to rename this variable